### DISCUSSION TITLE
Play Sound on First Message

### DISCUSSION THREAD

**Discussion by Nerixyz** [Chatterino/chatterino2#4449] [Ideas]
_Originally asked by @khancerberus in https://github.com/SevenTV/chatterino7/discussions/198._

> If there is such a functionality in the current versions, could you explain me how to do it? if not, it would be good to develop it

This isn't possible right now, but in theory this could work.

Realistically, you'd probably want to restrict the sound to some channels. This requires a lot more effort (maybe something plugins could solve?).

**Comment by Felanbird** [comment]
cc #3374
